package.json

```diff
@@ -45,8 +45,9 @@
   "dependencies": {
     "canvas-toBlob": "^1.0.0",
     "codemirror": "https://github.com/codemirror/CodeMirror/archive/v2.38.tar.gz",
+    "es6-promise": "^4.1.0",
```

Comment:
don't we already have https://github.com/tyrasd/overpass-turbo/blob/46a218845b98a0c8e069e5307840c411e451887e/package.json#L61?